add-ons/fabric8/README.md

```diff
@@ -33,7 +33,7 @@ We currently recommend you use at least 5Gb of RAM in your minishift VM and toda
 
 So start minishift using something like this:
 
-    $ minishift start --memory=5000 --cpus=5 --disk-size=50g
+    $ minishift start --memory=5120 --cpus=5 --disk-size=50g
```

Comment:
I don't think this looks nice. Does 5G or 5GB not work? If not let's wait until https://github.com/minishift/minishift/issues/1068 is resolved.

Reply:
RAM fix has been merged into Minishift and released as 1.5.0